cmd/tailscale/cli/web.html

```diff
@@ -86,14 +86,28 @@ <h3 class="text-3xl font-semibold mb-3">Log in</h3>
 	<div class="mb-4">
 		<p>You are connected! Access this device over Tailscale using the device name or IP address above.</p>
 	</div>
-	<a href="#" class="mb-4 link font-medium js-loginButton" target="_blank">Reauthenticate</a>
+	<a href="#" class="mb-4 js-advertiseExitNode">
+		{{if .AdvertiseExitNode}}
+		<button class="button button-red button-medium" id="enabled">Stop advertising Exit Node</button>
+		{{else}}
+		<button class="button button-blue button-medium" id="enabled">Advertise Exit Node</button>
+		{{end}}
+	</a>
+	<div class="mb-4">
+		<a href="#" class="mb-4 link font-medium js-loginButton" target="_blank">Reauthenticate</a>
+	</div>
 	{{ end }}
 </main>
 <script>(function () {
-let loginButtons = document.querySelectorAll(".js-loginButton");
+const advertiseExitNode = {{.AdvertiseExitNode}};
 let fetchingUrl = false;
+var data = {
+	AdvertiseRoutes: "{{.AdvertiseRoutes}}",
```

Comment:
Maybe remove this as it's not being used?

Reply:
I tried this, but it involves unwinding a stack of other code and I'd rather not, as all of this is neatly in place to support advertising subnet routes. At this point all the "server" code is done and all that's needed on top of this PR is a UI.